packages/react/src/stories/ic-theme.stories.mdx

```diff
@@ -185,3 +189,197 @@ export const Controlled = () => {
+            width="24px"
+            fill="#000000"
+          >
+            <path d="M0 0h24v24H0V0z" fill="none" />
+            <path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm-5.5-2.5l7.51-3.49L17.5 6.5 9.99 9.99 6.5 17.5zm5.5-6.6c.61 0 1.1.49 1.1 1.1s-.49 1.1-1.1 1.1-1.1-.49-1.1-1.1.49-1.1 1.1-1.1z" />
+          </svg>
+        </IcNavigationButton>
+        <IcNavigationItem label="Navigation" href="/" slot="navigation" />
+        <IcNavigationItem label="Navigation" href="/" slot="navigation" />
+        <IcNavigationItem label="Navigation" href="/" slot="navigation" />
+        <IcNavigationItem
+          label="Navigation"
+          href="/"
+          selected="true"
+          slot="navigation"
+        />
+        <IcNavigationItem label="Navigation" href="/" slot="navigation" />
+      </IcTopNavigation>
+      <IcHero
+        heading="Hero heading"
+        subheading="Hero description. This is a Hero component, it should be used as a page heading."
+        secondary-heading="Secondary Heading"
+        secondary-subheading="This is a secondary description."
+        full-width
+      >
+        <div slot="interaction" style={{ display: "flex" }}>
+          <ic-text-field
+            placeholder="Filter display"
+            label="Filter display"
+            hide-label
+          ></ic-text-field>
+          <IcButton variant="primary" style={{ marginLeft: 16 }}>
+            Filter
+          </IcButton>
+        </div>
+        <IcButton variant="secondary" slot="interaction">
+          See all
+        </IcButton>
+        <ic-link
+          href="https://google.com"
+          slot="interaction"
+          style={{ marginTop: 10 }}
+        >
+          Help
+        </ic-link>
+      </IcHero>
+      <IcSectionContainer style={{ display: "flex", flexDirection: "column", gap: "8px"}}>
+        <IcCheckbox
+          value="valueName1"
+          label="Unselected / Default"
+        >
+        </IcCheckbox>
+        <IcTextField
+          label="What is your favourite coffee?"
+          required
+          placeholder="Placeholder"
+          helperText="Such as Arabica, Robusta or Liberica"
+        >
+        </IcTextField>
+        <IcAlert
+          heading="This alert uses a custom message slot"
+          dismissible="true"
+        >
+          <IcButton slot="action" variant="secondary">
+            Button
+          </IcButton>
+          <IcTypography variant="body" slot="message">
+            <p>
+              This is some text and
+              <ic-link href="/"> this is an inline link ic-link</ic-link>
+              within the text
+            </p>
+          </IcTypography>
+        </IcAlert>
+        <div style={{ border: "1px solid #ffffff", padding: "4px", margin: "4px"}}>
+          <span style={{color:"#ffffff"}}>Content in here should always be dark mode</span>
+           <IcTheme mode="dark">
```

Comment:
Would it be helpful to use the components to indicate that they are wrapped with dark-mode IcTheme? 
i.e. 'This IcCard is wrapped by IcTheme with dark-mode enabled'?